test/unit/lodash.spec.js

```diff
@@ -0,0 +1,721 @@
+            elem1 = params[0];
+            objects = [
+                { 'a': { 'b': _.constant(2) } },
+                { 'a': { 'b': _.constant(1) } }
+            ];
+            method = _.method.apply(_, params);
+            result = _.map(objects, method);
+        });
+        it('should verify params', function () {
+            expect(params).to.have.length(1);
+            expect(typeof elem1).to.eql('string');
+            expect(elem1).to.have.length(3);
+        });
+        it('should returns the new invoker function', function () {
+            expect(result).to.eql([2,1]);
+        });
+    });
+
+    describe('methodOf', function () {
+        var params,
+            array,
+            obj,
+            methodOf,
+            result;
+        before(function () {
+            array = _.times(3, _.constant);
+            params = datasets.methodOf(array);
+            obj = params[0];
+            methodOf = _.methodOf.apply(_, params);
+            result = _.map(['a[2]', 'c[0]'], methodOf);
+        });
+        it('should verify params', function () {
+            expect(params).to.have.length(1);
+            expect(obj instanceof Object).to.eql(true);
+            expect(Object.keys(obj)).to.have.length(3);
+        });
+        it('should returns the new invoker function', function () {
+            expect(result).to.eql([2,0]);
+        });
+    });
+
+    describe('noop', function () {
+        var params;
+        before(function(){
+            params = datasets.noop();
+        });
+        it('should check types', function(){
+            expect(params[0] instanceof Object).to.eql(true);
+        });
+        it('should check properties obj', function(){
+            var elem1 = params[0];
+            expect(elem1).to.have.property('number', 5);
+            expect(elem1).to.have.property('square', 25);
+        });
+        it('should return undefined', function(){
+            expect(_.noop.apply(_, params)).to.eql(undefined);
+        });
+    });
+
+    describe('nthArg', function () {
+        describe('nthArg1', function () {
+            var params;
+            before(function(){
+                params = datasets.nthArg1();
+            });
+            it('should check type', function(){
+                var elem1 = params[0];
+                expect(typeof elem1).to.eql('number');
+            });
+            it('should return nth argument', function(){
+                var fun = _.nthArg.apply(_, params);
+                expect(fun('a', 'b', 'c', 'd')).to.eql('d');
+            });
+        });
+        describe('nthArg2', function () {
+            var params;
+            before(function(){
+                params = datasets.nthArg2();
+            });
+            it('should chek type', function(){
+                var elem = params[0];
+                expect(typeof elem).to.eql('number');
+            });
+            it('should return nth argument', function(){
```

Comment:
Change description to more clear for example: "should return last argument". Add more expect to make sure that user don't achieve that goal in other way (if there is one assertion user can count elements and pass index)